packages/web-react/src/components/Alert/demo/AlertIcons.tsx

```diff
@@ -1,50 +1,25 @@
-// Because there is no `dist` directory during the CI run
-/* eslint-disable import/no-extraneous-dependencies, import/extensions, import/no-unresolved */
 import React from 'react';
-// eslint-disable-next-line @typescript-eslint/ban-ts-comment
-// @ts-ignore: No declaration file
-import icons from '@lmc-eu/spirit-icons/dist/icons';
-import { IconsProvider } from '../../../context';
 import Alert from '../Alert';
+import { ALERT_SUCCESS_CONTENT, ALERT_INFO_CONTENT, ALERT_WARNING_CONTENT, ALERT_DANGER_CONTENT } from './alertContent';
 
-// @see: https://github.com/storybookjs/storybook/issues/8104#issuecomment-932310244
-// eslint-disable-next-line @typescript-eslint/no-unused-vars
-const Story = (props: unknown) => (
-  <IconsProvider value={icons}>
-    <Alert color="success" iconName="profile" UNSAFE_className="mb-600">
-      We sent you an activation link to email <strong>spirit@lmc.eu</strong>. See{' '}
-      <a href="./" className="link-underlined">
```

Comment:
I think the link here is intentional here, so it should be preserved.

Reply:
Added back + added in twig